src/language-js/print/call-arguments.js

```diff
@@ -248,6 +249,14 @@ function shouldGroupFirstArg(args) {
   }
 
   const [firstArg, secondArg] = args;
+
+  if (
+    firstArg.type === "ModuleExpression" &&
+    secondArg.type === "ObjectExpression"
```

Comment:
How about making this check more specific so that it would accept only `{ type: "module" }`?

Reply:
Sounds good